guides/v2.0/contributor-guide/contributing_docs.md

```diff
@@ -8,26 +8,42 @@ github_link: contributor-guide/contributing_docs.md
 redirect_from: /guides/v2.0/extension-dev-guide/Contribute_edg.md
 ---
 
-Share your knowledge with the community by contributing to Magento DevDocs! You can contribute by creating an issue or pull request (PR) on our [devdocs](https://github.com/magento/devdocs){:target="\_blank"} Github repository. We welcome all types of contributions; from minor typo fixes to new topics.
+Share your knowledge with the community by contributing to Magento DevDocs! You can contribute by creating an issue or pull request (PR) on our [devdocs](https://github.com/magento/devdocs){:target="_blank"} Github repository. We welcome all types of contributions; from minor typo fixes to new topics. As you contribute PRs, you can also gain Contribution Points with Community Engineering.
 
 Magento's team of technical writers reviews all issues and pull requests on a regular basis, and we do our best to address all issues as soon as possible. Working through the backlog takes time, though, so we appreciate your patience.
 
-<div class="bs-callout bs-callout-tip" id="tip" markdown="1">
+<div class="bs-callout bs-callout-info" id="info" markdown="1">
 If you are not sure where to start contributing, you can review our list of [suggested topics]({{ page.baseurl }}/contributor-guide/contributing_docs_suggested.html).
 </div>
 
-If you write and contribute a full topic, we will put your name (or company's name) at the top of the page, right under the title, and link it to your blog or website! We will also add your picture and a link to your Github account on the <a href="{{ page.baseurl }}/contributor-guide/quarterly-contributors.html#top-recent-contributors">Top recent contributors</a> page.
+If you write and contribute a full topic, we will put your name (or company's name) at the top of the page, right under the title, and link it to your blog or website! We will also add your picture and a link to your Github account on the [Top recent contributors]({{ page.baseurl }}/contributor-guide/quarterly-contributors.html#top-recent-contributors) page.
```

Comment:
"(or your company's name)"

Reply:
Maybe remove comma after "page" - not sure it's needed.

Reply:
edited